src/server/controllers/ApiController.js

```diff
@@ -69,6 +70,15 @@ export default class ApiController {
       .catch(err => response(res, err.message, 404));
   }
 
+  static getUserTitles(req, res) {
+    Effects
+      .find({}, 'title')
+      .then(titles => res.status(200).json({
+        userPresets: titles,
+      }))
+      .catch(error => console.log(error));
```

Comment:
I thisnk it will be better to send error in result instead console.log()

Reply:
done)